src/methods.ts

```diff
@@ -269,6 +269,7 @@ export default class Methods {
 			if (currentCache.frontmatter) {
 				//@ts-expect-error, could return null so can't be assigned to current aliases,
 				// check for null is done later
+				metaObj.frontmatter = currentCache.frontmatter;
```

Comment:
I've moved line 272 two lines up because the `@ts-expect-error` is for `currentAliases`. I used the GitHub interface for it, you'd have to make sure that indentation matches/make the change locally and push it.